presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java

```diff
@@ -218,6 +222,7 @@ public QueryInfo(
         this.aggregateFunctions = aggregateFunctions;
         this.windowsFunctions = windowsFunctions;
         this.planCanonicalInfo = planCanonicalInfo == null ? ImmutableList.of() : planCanonicalInfo;
+        this.planIdNodeMap = planIdNodeMap == null ? ImmutableMap.of() : ImmutableMap.copyOf(planIdNodeMap);
```

Comment:
Please consider renaming this to `planNodeIdToPlanNodeMap`

Reply:
I think `planNodeIdToPlanNodeMap` may be too verbosed?